src/main/java/org/opentripplanner/framework/graphql/GraphQLUtils.java

```diff
@@ -0,0 +1,54 @@
+package org.opentripplanner.framework.graphql;
+
+import graphql.schema.DataFetchingEnvironment;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.Nonnull;
+import org.opentripplanner.framework.i18n.I18NString;
+
+public class GraphQLUtils {
+
+  public static String getTranslation(I18NString input, DataFetchingEnvironment environment) {
+    if (input == null) {
+      return null;
+    }
+    return input.toString(getLocale(environment));
+  }
+
+  public static Locale getLocale(DataFetchingEnvironment environment) {
+    return getLocale(environment, environment.getArgument("language"));
```

Comment:
In the Transmodel API, the language parameter is sometimes called "lang", in those cases we don't use these utils. Should we change the utils to support both arguments?

Reply:
Thomas has created a separate ticket for this.